documentation/docs/.vuepress/config.js

```diff
@@ -0,0 +1,42 @@
+import { defaultTheme } from '@vuepress/theme-default';
+
+import { defineUserConfig } from 'vuepress';
+import { searchPlugin } from '@vuepress/plugin-search';
+import { viteBundler } from '@vuepress/bundler-vite';
+
+export default defineUserConfig( {
+    base: '/alphagov-wordpress-theme/',
+    lang: 'en-US',
+    title: 'Alpha-Gov WordPress Theme',
+    description: 'Developer Documentation for Alpha-Gov WordPress Theme',
+    bundler: viteBundler( {} ),
+    theme: defaultTheme( {
+        logo: '/images/BCID_H_rgb_pos.png',
+        logoDark: '/images/BCID_H_rgb_rev.png',
```

Comment:
interesting! does the vuepress installation know when the page is being viewed in dark mode, and switch the image?

Reply:
it does :)